src/clients/sw_single.rb

```diff
@@ -623,26 +625,24 @@ def StartSWSingle
                 )
               end
               force_restart = true
-            elsif result == :online_search
+            when :online_search
               WFM.CallFunction("online_search", [:sw_single_mode])
               force_restart = true
-            elsif result == :webpin
+            when :webpin
```

Comment:
Webpin has been dropped long time ago, you can delete it completely.

Reply:
good. I always enjoy killing code.